### PR TITLE
server: mention cli-side --drain-wait in corresponding server-side settings

### PR DESCRIPTION
**Pull request by tbg**
See (internal) https://github.com/cockroachlabs/support/issues/1318.

Release note: None

### CODE REVIEW

**Comment by tbg** [comment]
TODO: also expand the message on the cli side when the timeout is hit to remind user of the cluster settings and how they interact.

**Review by knz — APPROVED** [review]
LGTM but I can also have a look if you want to do the cli-side error thing too.

**Comment by tbg** [comment]
Just pushed the cli-side thing, lmk if it looks good

File: pkg/cli/quit.go [hunk 1]
```diff
@@ -113,7 +113,8 @@ func doDrain(
 	})
 	if errors.HasType(err, (*contextutil.TimeoutError)(nil)) || grpcutil.IsTimeout(err) {
 		log.Infof(ctx, "drain timed out: %v", err)
-		err = errors.New("drain timeout")
+		err = errors.New("drain timeout, consider adjusting --drain-wait, especially under " +
```
Comment (knz) [inline pkg/cli/quit.go:116]: maybe try this:
```go
err = errors.WithHint(err, "Consider adjusting --drain-wait, especially under " +
"custom server.shutdown.{drain,query,lease_transfer}_wait cluster settings.")
```

(assuming the hint shows up in the command output)

Comment (tbg) [reply pkg/cli/quit.go:116]: It does not, unfortunately:

```
$ ./cockroach node drain --insecure --drain-wait=1ns
node is draining... 
ERROR: operation timed out.

operation "drain" timed out after 1ns (took 0s): error sending drain request: rpc error: code = DeadlineExceeded desc = context deadline exceeded
Failed running "node drain"
```

I will leave as is.

**Comment by tbg** [comment]
bors r=knz

**Pull request merged**